lib/arrowController.js

```diff
@@ -0,0 +1,190 @@
+import { fabric } from 'fabric';
+
+class Arrow {
+  constructor(options = {}) {
+    this._mousePosition = null;
```

Comment:
I'd like move away from underscore danglers. We don't use it in any of our other projects and it's against the airbnb style.

https://eslint.org/docs/rules/no-underscore-dangle